usr/share/rear/conf/default.conf

```diff
@@ -92,22 +92,28 @@
 # the rescue/recovery system ISO image (and perhaps even stores the backup archive).
 # The directory name of the working area is created in /usr/sbin/rear by calling
 #   mktemp -d -t rear.XXXXXXXXXXXXXXX
-# which usually results /tmp/rear.XXXXXXXXXXXXXXX or $TMPDIR/rear.XXXXXXXXXXXXXXX
-# the latter when the canonical Linux/Unix environment variable TMPDIR
-# is set in the environment where /usr/sbin/rear is called.
-# To have a specific working area directory prefix for Relax-and-Recover call
+# which results $TMPDIR/rear.XXXXXXXXXXXXXXX for ReaR's working area.
+# To have a specific working area directory prefix for Relax-and-Recover specify
```

Comment:
Only a nit, but why is it better to say `specify` than `call`, given that it is an executable command and not some data to "specify"? Maybe `execute`, if you don't like `call`?

Reply:
Fixed by
https://github.com/rear/rear/commit/e0823bc04135df51857ed070ce5c5965144fac6c